scipy/stats/__init__.py

```diff
@@ -237,6 +237,7 @@
    tsem              --
    variation         -- Coefficient of variation
    find_repeats
+   rankdata
```

Comment:
`rankdata` was in hypothesis tests. It is *used* in hypothesis tests, but so is `find_repeats`, and neither *are* hypothesis tests.
I think it's reasonable to keep it close to `find_repeats`, wherever that belongs.